BD-3120 Add clarity to billing tables (#7554)
braze-inc/braze-docs: _docs/_user_guide/administrative/app_settings/subscription_and_usage.md

![Drill to workspaces for billable data points][9]{: style="max-width:90%;"}

##### Workspace level data point usage by category

> Select the menu next to an app group name and then **Billable Data Points For APP_GROUP_NAME** to view the **Workspace Level Data Point Usage by Category** table. This table shows you data point counts for each category of data points. For example, you can see the number of data points driven by sessions and custom events. You can use this table to identify the categories of data points that are driving data point consumption for the workspace.

Reviewer: Some outdated usage of app group here

```suggestion
Select the menu next to a workspace name and then **Billable Data Points For WORKSPACE_NAME** to view the **Workspace Level Data Point Usage by Category** table. This table shows you data point counts for each category of data points. For example, you can see the number of data points driven by sessions and custom events. You can use this table to identify the categories of data points that are driving data point consumption for the workspace.
```
